meson.build

```diff
@@ -0,0 +1,32 @@
+project('icu', 'c', 'cpp',
+  version: '67.1',
+  meson_version: '>=0.55.99',
```

Comment:
Why is 0.55 not good enough anymore?

Reply:
That is what I tested it on. Lowered to 0.55 as it should work on it, the main requirement is `[provides]`, but that was already in 0.55.